lighthouse-cli/test/smokehouse/byte-efficiency/expectations.js

```diff
@@ -103,4 +103,29 @@ module.exports = [
       },
     },
   },
+  {
+    requestedUrl: 'http://localhost:10200/byte-efficiency/gzip.html',
+    finalUrl: 'http://localhost:10200/byte-efficiency/gzip.html',
+    audits: {
+      'network-requests': {
+        details: {
+          items: [
+            {
+              url: 'http://localhost:10200/byte-efficiency/gzip.html',
+            },
+            {
+              url: 'http://localhost:10200/byte-efficiency/script.js?gzip=1',
+              transferSize: 1136,
```

Comment:
are these stable across platforms, like the native zlib mac/linux is the same byte-for-byte as js and transfer size is always the same and all that?

seems like we could loosen the assertion a bit

Reply:
Good question, no idea. For the purposes of the regression a ballpark is good enough (less than 1500 i guess).

Looks like travis / appveyor got the same thing I did.

This [SO answer](https://stackoverflow.com/questions/26516369/zlib-gzip-produces-different-results-for-same-input-on-different-oses) perhaps suggests that we can rely on zlib being the same? Interestingly there's a byte that represents the OS used, but that wouldn't change the length. You've got me curious now so I'm gonna dig in way deeper than necessary :P

Reply:
Any findings from the 🐰 🕳 ?

Reply:
Just that it may change when Node upgrades zlib, but they haven't update it in years. Seems fine to punt.